XiEditor/EditView.swift

```diff
@@ -228,7 +183,7 @@ class EditView: NSView, NSTextInputClient {
             // probably want to move to using CTLineDraw instead of drawing the attributed string,
             // but that means drawing the selection highlight ourselves (which has other benefits).
             //attrString.drawAtPoint(NSPoint(x: x0, y: y - 13))
-            let y = dataSource.textMetrics.linespace * CGFloat(lineIx + 1);
+            let y = defaultStyle.linespace * CGFloat(lineIx + 1);
```

Comment:
I am not sure, but would consider it possible that different font weights have a different `linespace`, which is not taken into account here. I hope we can ignore it and just keep using the default style `linespace` 🤔